book.rb

```diff
@@ -0,0 +1,14 @@
+# Book object template
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author, :rentals
+
+  def add_rental(date, person)
+    @rentals << Rental.new(self, person, date)
```

Comment:
- [ ] You need to import the **`Rental`** class into this book file for this instantiation to work. Kindly do that.

  ![dante-rental-class](https://user-images.githubusercontent.com/36542986/157546559-037047d0-7f3e-4bc5-99f6-c34d6d3bda59.jpg)

Reply:
Sorry! I fixed it!